tensorflow/cc/gradients/math_grad_test.cc

```diff
@@ -503,6 +511,34 @@ TEST_F(CWiseUnaryGradTest, Angle) {
   TestCWiseGrad<complex64, float>(ANGLE, x_fn);
 }
 
+TEST_F(CWiseUnaryGradTest, Lgamma) {
+  auto x_fn = [this](const int i) {
+    return RV({-3.5, -2.5, -1.5, 1.0, 2.0, 3.5});
+  };
+  TestCWiseGrad<float, complex64>(LGAMMA, x_fn);
```

Comment:
Can you verify if these tests pass locally? They should have failed because of the mismatched types of the input/output.
 
`TestCWiseGrad<float, float>` - these two parameters indicate the incoming and outgoing types for the operator, and Lgamma on a real will also output a real.

Reply:
Oh, that's my question: I know how to test on python using bazel, could you tell me how to test a specific case on c++? Thanks.

Reply:
Thanks, I found it. Bazel can run all tests, cool.